### ISSUE TITLE
Segfault during scan with Jax 5.2 with Nvidia driver 570 and cuda 12.8

### ISSUE THREAD

**Issue by alik-git** [jax-ml/jax#27235]
### Description

placeholder 

### System info (python version, jaxlib version, accelerator, etc.)

(master) ~/repos/ksim_stuff/ksim/
$ python -c "import jax; jax.print_environment_info()"
jax:    0.5.2
jaxlib: 0.5.1
numpy:  2.2.4
python: 3.11.11 (main, Dec 11 2024, 16:28:39) [GCC 11.2.0]
device info: NVIDIA GeForce RTX 4090-1, 1 local devices"
process_count: 1
platform: uname_result(system='Linux', node='omega', release='6.8.0-52-generic', version='#53~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Wed Jan 15 19:18:46 UTC 2', machine='x86_64')

$ nvidia-smi
Tue Mar 18 16:26:55 2025       
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 570.124.06             Driver Version: 570.124.06     CUDA Version: 12.8     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA GeForce RTX 4090        Off |   00000000:01:00.0  On |                  Off |
|  0%   41C    P8             21W /  450W |    1492MiB /  24564MiB |     19%      Default |
|                                         |                        |                  N/A |
+-----------------------------------------+------------------------+----------------------+
                                                                                         
+-----------------------------------------------------------------------------------------+
| Processes:                                                                              |
|  GPU   GI   CI              PID   Type   Process name                        GPU Memory |
|        ID   ID                                                               Usage      |
|=========================================================================================|
|    0   N/A  N/A            2018      G   /usr/lib/xorg/Xorg                      565MiB |
|    0   N/A  N/A            2192      G   /usr/bin/gnome-shell                    116MiB |
|    0   N/A  N/A            3846      G   ...ess --variations-seed-version        228MiB |
|    0   N/A  N/A            6447      G   ...ersion=20250318-050124.533000        112MiB |
|    0   N/A  N/A           19994      C   python                                  390MiB |
+-----------------------------------------------------------------------------------------+

(ksim) ali@omega [04:26:55PM 18/03/2025]:
(master) ~/repos/ksim_stuff/ksim/
$

**Comment by hawkinsp** [comment]
I'm guessing this issue is stale, and there is not enough information here for me to try to reproduce. If this is still a problem, please file a new issue with runnable instructions to reproduce.
